python/adelphi/README.md

```diff
@@ -0,0 +1,129 @@
+# Adelphi tooling
+
+## Introduction
+A tool for interacting with the DataStax [Adelphi](https://github.com/datastax/adelphi) project.  This package provides the "adelphi" application which in turn provides the following features:
+
+* Extraction of schemas for one or more keyspaces from a running Cassandra cluster
+* Optionally anonymizing these schemas
+* Formatting these schemas as CQL statements or as JSON documents
+* Displaying these formatted schemas on standard out or writing them to the filesystem
+* Automate a workflow for contributing anonymized schemas to the public [Adelphi schema repository](https://github.com/datastax/adelphi-schemas)
+
+The anonymization process replaces all keyspace names, table names and table column names with a generic identifier.  You can use the "adelphi" application to extract, format and display schemas from your Cassandra clusters without contributing these schemas to the Adelphi project, and for this use case anonymization is not required.  Anonymization *is* required anytime you contribute schemas to the Adelphi project.  All the schemas in our repository are publicly visible so to avoid *any* possible leakage of proprietary information we can only accept schemas which have been anonymized.
```

Comment:
I'd highlight the last part about visibility with some code-block style or at least move it to a new paragraph.

Reply:
Hmmm, the content of that line fits in with the rest of that paragraph (in fact that paragraph was structured so that it would lead up to that statement) so I don't really want to decouple them.  But I think perhaps making just that sentence bold would satisfy both of us.  I'll give that a shot.